doc/classes/Dictionary.xml

```diff
@@ -4,16 +4,60 @@
 		Dictionary type.
 	</brief_description>
 	<description>
-		Dictionary type. Associative container which contains values referenced by unique keys. Dictionaries are always passed by reference.
+		Dictionary type. Associative container which contains values referenced by unique keys. Just like in real life each dictionary composes multiple pairs of unique keys and values. Each pair is created by joining them via colon [code]Key:Value[/code].
 		Erasing elements while iterating over them [b]is not supported[/b].
 		Creating a dictionary:
 		[codeblock]
-		var d = {4: 5, "A key": "A value", 28: [1, 2, 3]}
+		var my_dir = {} # Creates an empty dictionary
+		var points_dir = {"White" : 50,"Yellow" : 75, "Orange" : 100}
```

Comment:
Dictionary spacing is odd. Usually, they are written as `{key: value, key: value}`, not as `{key : value,key : value}`.